lib/boavizta.rb

```diff
@@ -50,7 +50,7 @@ def self.carbon_for_load(device, cpu_load)
         req.params['duration'] = 1
       end
     end
-    JSON.parse(response.body).dig(*%w[impacts gwp use value])
+    JSON.parse(response.body).dig(*%w[impacts gwp use value]) * 1000
```

Comment:
Yes. Given a bit more time on this it would've been another good opportunity to use a data migration, but in the meantime I can update the data on the public server myself. I did consider keeping the database units the same and doing the conversions elsewhere, but it seemed silly for our default unit to be one that we no longer use and to be constantly converting.

Reply:
I have since changed my mind on the data migration part - the latest version does indeed have a data migration.